plugin.video.pcloud-video-streaming/addon.py

```diff
@@ -153,7 +153,8 @@ def AuthenticateToPCloud():
 			oneFileOrFolderItem = oneItem["metadata"]
 		filename = oneFileOrFolderItem["name"]
 		if oneFileOrFolderItem["isfolder"]:
-			li = xbmcgui.ListItem(filename, iconImage='DefaultFolder.png')
+			li = xbmcgui.ListItem(filename)
+			li.setIconImage("DefaultFolder.png")
```

Comment:
a quick note in case you're not aware: this function has been removed in matrix. No problem in using it for helix-leia though.